avocado/utils/ssh.py

```diff
@@ -268,6 +269,8 @@ def copy_files(self, source, destination, recursive=False):
             options += " -r"
         options += f" {source} {destination}"
         try:
+            if self.cmd(f"test -d {path}").exit_status != 0:
+                self.cmd(f"mkdir -p {path}")
```

Comment:
What about to reuse `init_dir` from `avocado.utils.path` ?

Reply:
The init_dir method is used to execute the machine，check whether the peer machine has a path，I think the test command makes it easier to check if a path exists on the client machine,because we have wrapped functions that we can use but we need to use the find_command method to check if test and mkdir are available，This is my lack of thinking

Reply:
I see, thanks for explanation.